src/components/PrimaryPanes/SourcesTree.js

```diff
@@ -291,7 +300,7 @@ class SourcesTree extends Component {
       expanded,
       onExpand: (item, expandedState) => setExpandedState(expandedState),
       onCollapse: (item, expandedState) => setExpandedState(expandedState),
-      renderItem: this.renderItem
+      renderItem: this.renderItem.bind(this, sources)
```

Comment:
I think we can update this pattern a bit: because we already bind on line 79 (here[h]) we probably can just get the sources in `renderItem` w/ `const {sources} = this.props` and avoid extraData.

[h]:https://github.com/tohmjudson/debugger.html/blob/6437b275ba76784f9d7d3eb84767513924d79361/src/components/PrimaryPanes/SourcesTree.js#L79

Reply:
I like it, I try it out 👍

Reply:
Good catch, makes it much cleaner :)